### PR TITLE
Update UDMI schema naming

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

File: schemas/udmi/metadata.tests/errors.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     },
     "physical_tag": {
       "asset": {
-        "name": "US-SFO-XYY_AHU-A01_Noope!"
+        "site": "US-SFO-XYY_Noope!",
+        "name": "AHU-A01_extension11-option"
```
[the remark this answers is not in the document]
Comment (grafnu) [reply schemas/udmi/metadata.tests/errors.json:12]: Correct... intenttional error.

File: schemas/udmi/metadata.tests/example.json [hunk 2]
```diff
@@ -13,8 +13,12 @@
     "physical_tag": {
       "asset": {
         "guid": "bim://04aEp5ymD_$u5IxhJN2aGi",
-        "name": "US-SFO-XYY_AHU-1_extention11-optional"
+        "site": "US-SFO-XYY",
+        "name": "AHU-1"
       }
+    },
+    "aux": {
+      "suffix": "extention11-optional",
```
[the remark this answers is not in the document]
Comment (grafnu) [reply schemas/udmi/metadata.tests/example.json:21]: This is meant to be a standalone representation of the metadata for a device.  If the suffix is going to be printed on the QR code or other label, then I want it included here.  It's OK if it's represented elsewhere too -- these metadata files will likely be automatically generated anyway from some other source (e.g. assert registry).

**Comment by grafnu** [comment]
Fundamental about the suffix: is this the thing that is potentially going to be printed on the QR code label?  (not _in_ the QR code, but as english text)

**Comment by pisuke** [comment]
Yes, the suffix is intended to be printed on the physical label (as a transition workaround if needed) and not included in the QR code payload.

**Comment by grafnu** [comment]
Ok, then I think the metadata as I have it is proper.  Basically I'm
envisioning a tool that can be used to create / scan / whatever labels, so
it should have all the information in there. These are generated low-level
files, not meant to be "an authoritative source"...

**Pull request merged**